test/e2e/search_test.go

```diff
@@ -122,6 +122,11 @@ registries = ['{{.Host}}:{{.Port}}']`
 		search.WaitWithDefaultTimeout()
 		Expect(search.ExitCode()).To(Equal(0))
 		Expect(len(search.OutputToStringArray())).To(Equal(4))
+
+		search = podmanTest.Podman([]string{"search", "--limit", "30", "docker.io/alpine"})
+		search.WaitWithDefaultTimeout()
+		Expect(search.ExitCode()).To(Equal(0))
+		Expect(len(search.OutputToStringArray())).To(Equal(31))
```

Comment:
Could you add a test to verify that without --limit you get 25 lines returned?  Or is that in play already?

Reply:
fixed. Add another search under `--limit` section to check the number of lines.